src/array_api_extra/_apply.py

```diff
@@ -0,0 +1,351 @@
+"""Public API Functions."""
+
+# https://github.com/scikit-learn/scikit-learn/pull/27910#issuecomment-2568023972
```

Comment:
Out of scope: I think it's really unhealthy to unformally support Python 3.9, without any CI to back it.